lib/nodes/list_item.js

```diff
@@ -1,7 +1,7 @@
 export const name = "list_item";
 
 export const schema = {
-  content: "paragraph block*",
+  content: "paragraph",
```

Comment:
Surprised this didn't need an update to a test, because previously a lot of options would have been enabled still?